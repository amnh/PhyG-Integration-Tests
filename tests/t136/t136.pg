--build(distance, nj, block), inputting tcm (X,Y) and fasta file
set(seed:1634561640)
read(nucleotide:"chel-prealigned.fas", tcm:(2,1))
read(nucleotide:"chel_cox1aln.fasta", tcm:"sg1t4.mat")
build(distance, nj, block, eun, displaytrees:3)
report("t136.csv", data, overwrite)
report("t136.tre", newick, overwrite)