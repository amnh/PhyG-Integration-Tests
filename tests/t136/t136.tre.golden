(Alentus:301.5,(Chanbria:160.5,(((Americhernus:200.0,Aportus:158.5)HTU19:120.0,(Limulus:106.0,(Centruoides:121.5,Hadrurus:126.5)HTU18:148.0)HTU21:38.0)HTU20:54.0,((Gea:147.5,(Artemia:163.0,Thermobius:164.0)HTU24:73.5)HTU25:69.5,(Rhiphicephalus:172.5,(Vonones:118.0,(Pauroctonus:99.0,(Mastigoproctus:121.5,(Amblypygid:147.0,(Hypochilus:158.0,Thelichoris:136.0)HTU23:82.5)HTU31:83.5)HTU30:42.0)HTU29:53.0)HTU28:49.0)HTU27:52.5)HTU26:37.5)HTU22:49.0)HTU32:206.5)HTU17[2316.0];
(Alentus:301.5,(Chanbria:160.5,(((Americhernus:200.0,Aportus:158.5)HTU19:120.0,(Limulus:106.0,(Centruoides:121.5,Hadrurus:126.5)HTU18:148.0)HTU21:38.0)HTU20:54.0,((Gea:147.5,(Artemia:163.0,Thermobius:164.0)HTU24:73.5)HTU25:69.5,(Rhiphicephalus:172.5,(Vonones:118.0,(Pauroctonus:99.0,(Mastigoproctus:121.5,(Amblypygid:147.0,(Hypochilus:158.0,Thelichoris:136.0)HTU23:82.5)HTU31:83.5)HTU30:42.0)HTU29:53.0)HTU28:49.0)HTU27:52.5)HTU26:37.5)HTU22:49.0)HTU32:206.5)HTU17[2316.0];
(Alentus:288.0,(Chanbria:165.5,(((Centruoides:105.0,Hadrurus:143.5)HTU18:90.0,(Americhernus:201.5,Aportus:138.5)HTU19:165.5)HTU20:57.0,((Gea:149.5,(Artemia:156.5,Thermobius:167.5)HTU23:59.5)HTU24:54.5,(Rhiphicephalus:185.0,(Vonones:93.0,(Limulus:144.0,(Pauroctonus:110.0,(Mastigoproctus:122.0,(Amblypygid:153.5,(Hypochilus:173.0,Thelichoris:144.0)HTU22:80.0)HTU31:86.0)HTU30:41.0)HTU29:64.5)HTU28:52.0)HTU27:57.0)HTU26:75.0)HTU25:38.5)HTU21:50.5)HTU32:192.0)HTU17[2305.0];
